src/graph/backend/elyzor/target_machine.hpp

```diff
@@ -16,35 +16,20 @@
 #ifndef BACKEND_ELYZOR_TARGET_MACHINE_HPP
 #define BACKEND_ELYZOR_TARGET_MACHINE_HPP
 
-// An internal GC's functionality that provides machine flags,
-// assuming all the flags are true for now
-// #include "core/src/compiler/config/context.hpp"
-// #include "runtime/config.hpp"
+#include "cpu/x64/cpu_isa_traits.hpp"
 
 #define REQUIRE_AVX512_BEGIN \
-    if (true) {
-    // if (::dnnl::impl::graph::gc::get_default_context() \
-    //                 ->machine_.cpu_flags_.fAVX512F) {
+    if (dnnl::impl::cpu::x64::cpu().has(Xbyak::util::Cpu::tAVX512F)) {
```

Comment:
gc internals: [check 16th bit of info[1] ](https://github.com/oneapi-src/oneDNN/blob/1344a14b4160d62a18dcc447a2c0f122f87b5b03/src/graph/backend/graph_compiler/core/src/runtime/target_machine.cpp#L155-L156)
xbyak utils: [check 16th bit of EBX (aka info[1]) ](https://github.com/oneapi-src/oneDNN/blob/1344a14b4160d62a18dcc447a2c0f122f87b5b03/src/cpu/x64/xbyak/xbyak_util.h#L619)